samples/apps/agent-id-sample/frontend/src/styles.css

```
@@ -0,0 +1,2712 @@
+  gap: 18px;
+}
+
+.booking-confirmed-panel {
+  width: min(1180px, calc(100% - 40px));
+  max-width: 1180px;
+  margin: 22px auto 0;
+}
+
+.booking-flight-widget {
+  --booking-route-code-size: clamp(1.7rem, 4vw, 3rem);
+  display: grid;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)